Cargo.toml

```
@@ -36,6 +36,7 @@ required-features = ["anyhow", "backtrace", "chrono", "uuid"]
 [features]
 default = []
 catch-unwind = ["pin-project"]
+prefer-zero-copy = []
```

lets just go with `zero-copy` here.
```suggestion
zero-copy = []
```